.changeset/shiny-brooms-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ai-sdk/orchestration': minor
+---
+
+[Improvement] Add convenience function for creating document grounding configuration
```

Comment:
[Improvement] Add convenience function to create document grounding configuration